views/modules.haml

```diff
@@ -5,21 +5,35 @@
       %h1 Redis Modules
 
       %p
-        This is a list of Redis modules, for Redis v4.0 or greater, ordered by Github stars. This list contains two set of modules: modules under an OSI approved license, and modules that are under some proprietary license. Non OSI modules are clearly flagged as not open source. Also to have the source code hosted at Github is currently mandatory. To add your module here please send a pull request for <a href="https://github.com/redis/redis-doc/blob/master/modules.json">the modules.json file</a> in the <a href="https://github.com/redis/redis-doc">Redis-doc</a> repository. <a href="https://redislabs.com/community/redis-modules-hub/">More information about Redis modules can be found here</a>.
+        This is a list of Redis modules, for Redis v4.0 or greater, ordered by activity and Github stars.
+
+      %p
+        Modules with some activity in their GitHub repository within the last six months are marked with a <i class="fa fa-smile-o icon"></i>.
+
+      %p
+        This list contains two types of modules: modules under an OSI approved license, and modules that are under some proprietary license. Non-OSI modules are clearly flagged as not open source. Also to have the source code hosted at Github is currently mandatory.
```

Comment:
The RSL and pHash modules aren't really "clearly marked", they have the license listed but I would argue that is not un-ambiguous.